charts/prometheus-statsd-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus-statsd-exporter
 description: A Helm chart for prometheus stats-exporter
-version: 0.4.2
+version: 0.4.3
```

Comment:
new feature should raise minor version.

docher image could be updated to 0.22.5 too.

Reply:
Increased minor version and updated docker image to 0.22.7 (latest and working with current chart).